### DISCUSSION TITLE
How to call the mergeCells() method

### DISCUSSION THREAD

**Discussion by acubick** [theoephraim/node-google-spreadsheet#616] [Q&A]
Help me understand how to call the mergeCells() method
Hey guys.
I can't call the .mergeCells() method
Already tried different options.
I see that in the documentation and in the code it is available. 
Can you show me an example of my code what I'm doing wrong?
Thanks.

I have this code:
  ` const doc = new GoogleSpreadsheet(<GoogleSpreadsheet_ID>)`

  ` await doc.loadInfo()`

   `const sheetSource = doc.sheetsByIndex[0]`

   ` await sheetSource.updateProperties({
        title: 'General report'
    })`  
   ` await sheetSource.loadCells({
        startRowIndex: 5, endRowIndex: 6, startColumnIndex:0, endColumnIndex: 5
    })` 
   `  const cellA1 = await sheetSource.getCell(1, 2)`  
    `await sheetSource.mergeCells() //  <=  this is not working `
<img width="740" alt="Screenshot 2023-04-01 at 13 32 11" src="https://user-images.githubusercontent.com/43810325/229281064-0ccb3b95-e522-4d9b-bb73-f423e63b5c30.png">

**Comment by theoephraim** [comment]
I think this was just an issue the types (which were maintained separately) were outdated. The library is now rewritten in Typescript so this wont happen again!
